src/autogluon/bench/scripts/generate_cloud_configs.py

```diff
@@ -0,0 +1,144 @@
+import typer
+import yaml
+
+app = typer.Typer()
+
+
+@app.command()
+def generate_cloud_config(
+    module: str = typer.Option(..., help="The module to generate config for: 'multimodal' or 'tabular'"),
+    root_dir: str = typer.Option(".ag_bench_runs", help="Root directory (optional, default = '.ag_bench_runs')"),
+    benchmark_name: str = typer.Option("ag_bench", help="Benchmark name (optional, default = 'ag_bench')"),
+    cdk_deploy_account: str = typer.Option(..., help="CDK deploy account"),
+    cdk_deploy_region: str = typer.Option(..., help="CDK deploy region"),
+    prefix: str = typer.Option(
+        "ag-bench", help="Prefix used to identify infra resources (optional, default = 'ag-bench')"
+    ),
+    metrics_bucket: str = typer.Option(..., help="Name of the metrics bucket (required)"),
+    data_bucket: str = typer.Option(None, help="S3 bucket to download private datasets (optional)"),
+    vpc_name: str = typer.Option(None, help="Existing VPC name (optional, default: create a new VPC)"),
+    max_machine_num: int = typer.Option(
+        None, help="Maximum number of machines for AWS Batch (optional, default = 20)"
+    ),
+    block_device_volume: int = typer.Option(
+        None, help="Block device volume for EC2 instances (optional, default = 100)"
+    ),
+    reserved_memory_size: int = typer.Option(
+        None, help="Reserved memory size for Docker container (optional, default = 15000)"
+    ),
+    instance: str = typer.Option(None, help="EC2 Instance type (optional, default = 'g4dn.2xlarge')"),
+    git_uri_branch: str = typer.Option("", help="AutoGluon MultiModal git_uri#branch (comma-separated)"),
```

Comment:
Is it comma-separated or pound separated? Your example says #

Reply:
Sorry it's confusing. It should be `git_uri#branch1,git_uri#branch2,...`. I'll update the note.

Reply:
So we can pass in multiple branches? What would happen in this case? Will we run the same benchmark on each of the custom branches?

Reply:
That's right. Basically we run benchmarks on the combination of setups defined in cloud configs.